app/assets/javascripts/discourse/controllers/review-index.js.es6

```diff
@@ -85,6 +85,47 @@ export default Controller.extend({
     return filtersExpanded ? "chevron-up" : "chevron-down";
   },
 
+  @discourseComputed(
+    "type",
+    "priority",
+    "status",
+    "category_id",
+    "username",
+    "from_date",
+    "to_date",
+    "sort_order",
+    "additional_filters"
+  )
+  anyFilters(
+    type,
+    priority,
+    status,
+    category_id,
+    username,
+    from_date,
+    to_date,
+    sort_order,
+    additional_filters
+  ) {
+    return (
+      type != null ||
+      priority !== "low" ||
+      status !== "pending" ||
+      category_id !== null ||
+      username !== "" ||
+      from_date !== null ||
+      to_date !== null ||
+      sort_order !== "priority" ||
+      (additional_filters !== null &&
+        JSON.stringify(additional_filters) !== "{}")
+    );
+  },
+
+  @discourseComputed("anyFilters")
```

Comment:
Can probably be replaced by: https://api.emberjs.com/ember/3.12/classes/@ember%2Fobject%2Fcomputed/methods/not?anchor=not